frontend/src/views/Dashboard/Dashboard.js

```diff
@@ -6,53 +6,43 @@ import {_getDashboardPath, DASHBOARD_PATHS} from "../../_routes";
 import {Link} from "react-router-dom";
 import InfoCard from "../../core/components/InfoCard/InfoCard";
 import SortableTable from "../../core/components/SortableTable";
-import {NETWORK_TABLE_COLUMNS} from "../../constants";
-
-// TODO: Remove dummy data and connect to backend.
-const chains = [
-  {
-    name: "Aion Testnet Network",
-    netID: 32,
-    hash: "a969144c864bd87a92e974f11aca9d964fb84cf5fb67bcc6583fe91a407a9309",
-  },
-  {
-    name: "Aion Mainnet Network",
-    netID: 256,
-    hash: "8ef9a7c67f6f8ad14f82c1f340963951245f912f037a7087f3f2d2f9f9ee38a8",
-  },
-  {
-    name: "Ethereum Mainnet Network",
-    netID: 1,
-    hash: "0de3141aec1e69aea9d45d9156269b81a3ab4ead314fbf45a8007063879e743b",
-  },
-  {
-    name: "Ethereum Rinkeby Network",
-    netID: 4,
-    hash: "8cf7f8799c5b30d36c86d18f0f4ca041cf1803e0414ed9e9fd3a19ba2f0938ff",
-  },
-  {
-    name: "Ethereum Ropsten Network",
-    netID: 3,
-    hash: "10d1290eee169e3970afb106fe5417a11b81676ce1e2119a0292df29f0445d30",
-  },
-  {
-    name: "Ethereum Goerli Network",
-    netID: 5,
-    hash: "4ae7539e01ad2c42528b6a697f118a3535e404fe65999b2c6fee506465390367",
-  },
-];
+import {NETWORK_TABLE_COLUMNS, APPLICATIONS_LIMIT} from "../../constants";
+import NetworkService from "../../core/services/PocketNetworkService";
+import Loader from "../../core/components/Loader";
+import ApplicationService from "../../core/services/PocketApplicationService";
 
 class Dashboard extends Component {
   constructor(props, context) {
     super(props, context);
 
     this.state = {
       alert: true,
+      loading: true,
+      chains: [],
+      userApps: [],
     };
   }
 
+  async componentDidMount() {
+    const userEmail = UserService.getUserInfo().email;
+    const networkHashes = [
+      "a969144c864bd87a92e974f11aca9d964fb84cf5fb67bcc6583fe91a407a9309",
+      "8ef9a7c67f6f8ad14f82c1f340963951245f912f037a7087f3f2d2f9f9ee38a8",
+      "0de3141aec1e69aea9d45d9156269b81a3ab4ead314fbf45a8007063879e743b",
+      "d9d77bce50d80e70026bd240fb0759f08aab7aee63d0a6d98c545f2b5ae0a0b8",
+    ];
```

Comment:
We have a backend method to retrieve al network chains  data

Reply:
Yeah, fixed it on my latest commit